lib/resources/tasks/lint.ts

```diff
@@ -0,0 +1,11 @@
+import * as gulp from 'gulp';
+import tslint from 'gulp-tslint';
+import * as project from '../aurelia.json';
+
+export default function lint() {
+  return gulp.src([project.transpiler.source])
```

Comment:
Nope this works ok. The reason is that I personally prefer to lint my unit tests as well so I'd typically add them here too. Not sure though if that counts for the masses. What would you say?